src/Messenger/Clients/Sns.php

```diff
@@ -67,6 +73,18 @@ private function getNameFromArn($arn)
         return $name;
     }
 
+    private function getDataToPublish($topic, $message)
+    {
+        //Up to 256KB of Unicode text.
+        $messageEncoded = json_encode($message);
+
+        return [
+            'TopicArn' => $this->getArnFromName($topic),
```

Comment:
Why did u name vars in PascalCase?

Reply:
oo srry! I see why below

Reply:
The AWS SDK always use pascal case for its settings (as you can see, this method is only called on the publish method)